antora-ui-camel/src/img/brand-logos.svg

```diff
@@ -20,4 +20,9 @@
     <desc>Chat on Zulip</desc>
     <path d="m 131.6034,4 5.79516,12.766113 h 23.09009 L 131.44616,46.136718 137.94714,60 h 41.74537 L 173.0343,46.136718 H 150.95764 L 179.22253,16.608887 173.11291,4 Z" />
   </symbol>
+  <symbol id="linkedin" viewBox="192 0 64 64">
+    <title>Linkedin</title>
+    <desc>Apache Camel group on Linkedin</desc>
+    <path d="m 131.6034,4 5.79516,12.766113 h 23.09009 L 131.44616,46.136718 137.94714,60 h 41.74537 L 173.0343,46.136718 H 150.95764 L 179.22253,16.608887 173.11291,4 Z" />
```

Comment:
This seems to be copy-pasted icon of Zulip, there are icons available from [LinkedIn](https://brand.linkedin.com/downloads), though they seem to be in PNG and we want SVG icons here. We should also review that we comply with their [policies](https://brand.linkedin.com/policies).

Reply:
Yes its a copy/paste of that icon, to ensure that a icon is shown. As I could not get it to show the linkedin icon.

Reply:
Since the icon is not shown on your end too, it may be something wrong with the preview mode, and that its shown on the real website.

Reply:
Preview is correct. The copy pasted Zulip icon has geometry that is outside of the viewbox, even if it was shown it would be confusing to the users to see two icons for Zulip.

Reply:
Okay so the icon works - and we can replace it with an actual linkedin icon.